pkgs/by-name/ca/caddy/package.nix

```
@@ -77,9 +77,10 @@ buildGoModule {
         command = "${caddy}/bin/caddy version";
         package = caddy;
       };
+      acme-integration = nixosTests.acme.caddy;
     };
     withPlugins = callPackage ./plugins.nix { inherit caddy; };
-  };
+};
```

Looks like an accidental indent change:
```suggestion
  };
```